annotator_store/models.py

```diff
@@ -10,25 +10,29 @@
 from django.contrib.auth.models import Group, User
 from django.utils.html import format_html
 from jsonfield import JSONField
-try:
-    import guardian
-    from guardian.shortcuts import assign_perm, get_objects_for_user, \
-        get_objects_for_group, get_perms_for_model, get_perms
-    from guardian.models import UserObjectPermission, GroupObjectPermission
-except ImportError:
-    guardian = None
-import six
-
 
+# get system logger for notifying test errors/warnings
 logger = logging.getLogger(__name__)
-
+#import settings to read ANNOTATION_OBJECT_PERMISSIONS flag
+from django.conf import settings
+#get per-object permission flag
+ANNOTATION_OBJECT_PERMISSIONS = getattr(settings, 'ANNOTATION_OBJECT_PERMISSIONS',False)
+# default guardian installed flag value is false
+guardian = False
+# check if per-object permission is enabled
+if ANNOTATION_OBJECT_PERMISSIONS:
+    try:
+        import guardian
+        from guardian.shortcuts import assign_perm, get_objects_for_user, \
+            get_objects_for_group, get_perms_for_model, get_perms
+        from guardian.models import UserObjectPermission, GroupObjectPermission
+    except ImportError:
+        logger.warning("Guardian must be installed for per-annotation permission!")
```

Comment:
Actually, is a warning sufficient here?  It seems like things will be pretty broken if object permissions are requested but guardian is not installed.  What if we drop the try/except and just let the import exception error act as our warning?

Reply:
It seems reasonable to me. I'm working on it.